ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-stable-4.15-upgrade-from-stable-4.15.yaml

```diff
@@ -74,17 +74,16 @@ tests:
     test:
     - chain: openshift-upgrade-qe-sanity-disconnected
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private
-- as: azure-ipi-proxy-tp-f28
+- as: gcp-ipi-proxy-tp-f28
```

Comment:
btw, the jobs are ordered by alphabet, shall we update the job in a right place?

Reply:
@jianlinliu could you share more about the context? I wonder if the job order controlled by some tools, or just by human beings.

Reply:
by human beings.